### DISCUSSION TITLE
Site is Live!

### DISCUSSION THREAD

**Discussion by curtisma** [mosaic-ic/mosaic-ic.github.io#5] [Announcements]
The site is now live at [https://www.mosaic-ic.org/](https://www.google.com/url?q=https://www.mosaic-ic.org/&sa=D&source=docs&ust=1662059034291938&usg=AOvVaw1IXEU7cpifBDcRuh3BEaSc)

@Lawrence-lugs, @mirjavm

**Comment by mirjavm** [comment]
Thx a lot Curtis 😊

Dr.​​

Mirjana

Videnovic‑Misic
Principal Scientist
Millimeter Wave Technologies
Intelligent Wireless Systems
M: +43 664 3547 302<tel:+43%20664%203547%20302>


From: Curtis Mayberry ***@***.***>
Sent: Thursday, September 1, 2022 8:08 PM
To: mosaic-ic/mosaic-ic.github.io ***@***.***>
Cc: Videnovic-Misic Mirjana ***@***.***>; Mention ***@***.***>
Subject: [mosaic-ic/mosaic-ic.github.io] Site is Live! (Discussion #5)


The site is now live at https://www.mosaic-ic.org/<https://eur04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.google.com%2Furl%3Fq%3Dhttps%3A%2F%2Fwww.mosaic-ic.org%2F%26sa%3DD%26source%3Ddocs%26ust%3D1662059034291938%26usg%3DAOvVaw1IXEU7cpifBDcRuh3BEaSc&data=05%7C01%7Cmirjana.videnovic-misic%40silicon-austria.com%7C631cccc62b6a414876c208da8c44de3e%7Ce0770667c71841db8e5ca9362ddf5914%7C0%7C0%7C637976524662682724%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=WCi%2BRy4yQQE5mGXoFT2rT2yAu6%2FO%2FylucblmCRWGCF0%3D&reserved=0>

@Lawrence-lugs<https://eur04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FLawrence-lugs&data=05%7C01%7Cmirjana.videnovic-misic%40silicon-austria.com%7C631cccc62b6a414876c208da8c44de3e%7Ce0770667c71841db8e5ca9362ddf5914%7C0%7C0%7C637976524662682724%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=DeMK4WZTFAMvreVW77hNFSFJkvp6Ng33L%2BmZMb%2FqCwA%3D&reserved=0>, @mirjavm<https://eur04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fmirjavm&data=05%7C01%7Cmirjana.videnovic-misic%40silicon-austria.com%7C631cccc62b6a414876c208da8c44de3e%7Ce0770667c71841db8e5ca9362ddf5914%7C0%7C0%7C637976524662682724%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=6s7ILKN%2BeXWiiXb%2B42AbKkJyeTzM5M%2B6XHuWc5XAac8%3D&reserved=0>
